### PR TITLE
Fix chart speed

### PR DESCRIPTION
**Pull request by robinschmid**
Fix chart speed, notify false, fix multiple tasks per chrom spec
### ChromatogramAndSpectraVisualizer
- every click in any chart was starting 2 tasks to update each plot - 4 in total
- added a small delay to simlify it
- set notify to false to only update the charts once everything is done

### CODE REVIEW

**Comment by SteffenHeu** [comment]
I noticed it before, do you know why the click event is even triggered twice in the first place?

File: src/main/java/io/github/mzmine/modules/visualization/spectra/simplespectra/SpectraVisualizerTab.java [hunk 1]
```diff
@@ -396,10 +397,14 @@ public void loadRawData(Scan scan) {
...
+    spectrumPlot.setNotifyChange(true);
+    spectrumPlot.fireChangeEvent();
   }
 
   public void loadPeaks(FeatureList selectedPeakList) {
-
+    spectrumPlot.setNotifyChange(false);
     spectrumPlot.removePeakListDataSets();
 
     if (selectedPeakList == null) {
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/modules/visualization/spectra/simplespectra/SpectraVisualizerTab.java:410]: if you set the notify to false and exit here, we will not get updates later on.

**Review by SteffenHeu — CHANGES_REQUESTED** [review]
Will also have to manually test it, but the changes look good so far, only one change needed.

File: src/main/java/io/github/mzmine/modules/visualization/spectra/simplespectra/SpectraVisualizerTab.java (hunk 1 above)
Comment (robinschmid) [reply src/main/java/io/github/mzmine/modules/visualization/spectra/simplespectra/SpectraVisualizerTab.java]: Done. Changed all to a try - finally

**Comment by SteffenHeu** [comment]
is it ready for final review?

**Comment by robinschmid** [comment]
Yes.

**Review by SteffenHeu — COMMENTED** [review]
will do some testing now

**Comment by SteffenHeu** [comment]
looks good

**Pull request merged**